query/stdlib/influxdata/influxdb/from_test.go

```diff
@@ -335,23 +335,33 @@ func TestFromRangeRule(t *testing.T) {
 
 	for _, tc := range tests {
 		tc := tc
-		t.Run(tc.Name, func(t *testing.T) {
-			t.Parallel()
-			plantest.RuleTestHelper(t, &tc)
-		})
+		if tc.Name == "cannot push range into from" {
```

Comment:
Why just in this case?

Reply:
Oops, that was temporary code to debug an issue.  I'll remove it.